components/layout_2020/table/construct.rs

```diff
@@ -159,3 +305,41 @@ where
         // do something?
     }
 }
+
+impl<'a, 'builder, 'dom, Node> TableRowBuilder<'a, 'builder, Node>
+where
+    Node: NodeExt<'dom>,
+{
+    fn consume_rowspans(&mut self) {
+        loop {
```

Comment:
nit: could this be `while let Some(...) = ...`?

Reply:
In later code I'm using the `current_x()` variable that's initialized outside of the if let.